### PR TITLE
Add integration specs

### PR DESCRIPTION
**Pull request by NatiDeme**
#Changes in PR:
- No linter errors
- Implemented integration specs
- Used Capybara to write integration tests

### CODE REVIEW

File: spec/views/posts/index.html.erb_spec.rb [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
+      expect(page).to have_content 'test'
+      expect(page).to have_content 'test'
+    end
+    it 'shows the Number of Likes' do
+      expect(page).to have_content('likes')
+    end
+    it 'shows Pagination' do
+      expect(page).to have_content('Pagination')
+    end
+  end
```
Comment (mateo951) [inline spec/views/posts/index.html.erb_spec.rb]: Please be sure to add a test that asserts you can see the number of posts the user has written as it is described in the requirements🤓

File: spec/views/posts/show.html.erb_spec.rb [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+    visit user_posts_path(@user1.id, @post1.id)
+  end
+  describe 'post show page' do
+    it 'shows the post title' do
+      expect(page).to have_content('First Post')
+    end
+    it 'shows the post body' do
+      expect(page).to have_content('Hello')
+    end
+  end
```
Comment (mateo951) [inline spec/views/posts/show.html.erb_spec.rb]: You still need to work on a couple of tests for your posts show but you are almost there! Kindly work on the following: 

* A Test that asserts you can see who wrote the post.
* A Test that asserts you can see how many comments it has.
* A Testthat asserts you can see how many likes it has.
* A Testthat asserts you can see the username of each commentor.
* A Testthat asserts you can see the comment each commentor left.

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi Team, 

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by rustam-tolipov — APPROVED** [review]
## Hello @NatiDeme👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**